css/alx-dark.css

```diff
@@ -884,16 +884,24 @@ body.dark .panel .list-group-item.in_second_deadline {
 body.dark .panel .list-group-item.in_second_deadline a {
   color: var(--col-second-deadline-link) !important;
 }
+
+body.dark .panel .list-group-item.in_second_deadline a:hover {
+  color: var(--col-light-400) !important;
+}
```

Comment:
I think this rule should go to the pending logs styles section on line 893. Just so we can spend less time searching for rules for components